community/kernel/src/main/java/org/neo4j/kernel/impl/storemigration/participant/StoreMigrator.java

```diff
@@ -520,6 +582,23 @@ private void prepareBatchImportMigration( File storeDir, File migrationDir, Reco
         }
     }
 
+    private void createStore( File migrationDir, RecordFormats newFormat )
+    {
+        StoreFactory storeFactory = new StoreFactory( new File( migrationDir.getPath() ), pageCache, fileSystem,
+                newFormat,
+                NullLogProvider.getInstance() );
+        try ( NeoStores neoStores = storeFactory.openAllNeoStores( true ) )
+        {
+            neoStores.getMetaDataStore();
+            neoStores.getLabelTokenStore();
+            neoStores.getNodeStore();
+            neoStores.getPropertyStore();
+            neoStores.getRelationshipGroupStore();
+            neoStores.getRelationshipStore();
+            neoStores.getSchemaStore();
```

Comment:
Are all of these calls here to make us fail fast if some store is present? In that case 👍